src/BrazeKit-dev.js

```diff
@@ -364,6 +373,7 @@ var constructor = function () {
     /**************************/
     function processEvent(event) {
         var reportEvent = false;
+        maybeSetConsentBeforeEventLogged(event);
```

Comment:
Would it be better to have 
`var eventConsentState = getEventConsentState(event.ConsentState)` 
above that and then pass `eventConsentState` to the `maybeSetConsentBeforeEventLogged`?
Similar to what I did in the google kits

Reply:
the reason I don't do that is because there is an if statement for why you may not want to even get the `eventConsentState` at all.  see my comment below related to this.  I don' thave an issue with Google being that way (as my comment probably applies there as well), but no need to go back and change it 4x.